src/content/releases/_index.en.md

```diff
@@ -11,137 +11,141 @@ weight = 30
 
-> This release has focused on stability, bugfixes and making [Lighthouse](https://github.com/submariner-io/lighthouse) available as
+> This release has focused on stability, bugfixes and making [Lighthouse](https://github.com/submariner-io/lighthouse) available as a
 > developer preview via `subctl` deployments.
 
-* Several bugfixes and enhancements around HA failover (#346, #348, #332)
-* Migrated to Daemonsets for submariner gateway deployment
-* Added support for hostNetwork to remote pod/service connectivity (#288)
-* Auto detection and configuration of MTU for vx-submariner, jumbo frames support (#301)
-* Support for updated strongswan (#288)
-* Better iptables detection for some hosts (#227)
-
-> `subctl` and the submariner Operator have the following improvements
-
-* support to verify-connectivity between two connected clusters
-* deployment of submariner gateways based in daemonsets instead of deployments
-* renaming submariner pods to "submariner-gateway" pods for clarity
-* print version details on crash (`subctl`)
-* stop storing IPSEC key on broker during deploy-broker, now it's only contained into the .subm file
-* version command for `subctl`
-* nicer spinners during deployment (thanks to kind)
+* Several bugfixes and enhancements around HA failover ([submariner#346](https://github.com/submariner-io/submariner/issues/346),
+  [submariner#348](https://github.com/submariner-io/submariner/pull/348),
+  [submariner#332](https://github.com/submariner-io/submariner/pull/332)).
+* Migrated to DaemonSets for Submariner gateway deployment.
+* Added support for hostNetwork to remote Pod/Service connectivity ([submariner#298](https://github.com/submariner-io/submariner/issues/298)).
+* Auto detection and configuration of MTU for vx-submariner, jumbo frames support
+  ([submariner#301](https://github.com/submariner-io/submariner/issues/301)).
+* Support for updated strongSwan ([submariner#288](https://github.com/submariner-io/submariner/issues/288)).
+* Better iptables detection for some hosts ([submariner#227](https://github.com/submariner-io/submariner/pull/227)).
+
+> `subctl` and the Submariner Operator have the following improvements:
+
+* Support for `verify-connectivity` checks between two connected clusters.
+* Deployment of Submariner gateways based on `DaemonSet` instead of `Deployment`.
+* Rename `submariner` Pods to `submariner-gateway` Pods for clarity.
+* Print version details on crash (`subctl`).
+* Stop storing IPSEC key on Broker during `deploy-broker`, now it's only contained into the `.subm` file.
+* Version command for `subctl`.
+* Nicer spinners during deployment (thanks to kind).
 
 ## v0.0.3 -- KubeCon NA 2019
 
-Submariner has been greatly enhanced to allow Operators to deploy into Kubernetes clusters without the necessity for layer-2 adjacency for
+Submariner has been greatly enhanced to allow Operators to deploy into Kubernetes clusters without the necessity for Layer 2 adjacency for
```

Comment:
```suggestion
Submariner has been greatly enhanced to allow operators to deploy into Kubernetes clusters without the necessity for Layer 2 adjacency for
```

Reply:
(here we are referring to the Kubernetes operators/adminstrators and not OLM)

Reply:
Maybe "administrators" would be more clear, given that we also built an Operator in that same time period?

Reply:
@dfarrell07 yeah, administrators would work.